tasks/post-deploy/04-clean-duplicate-postmeta.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Remove duplicate postmeta records from DB
+wp p4-gblocks remove_duplicate_postmeta
```

Comment:
It would be nice if the log output also contained the result of `Duplicated_Post_Meta::detect`. That way we can see which post_meta keys were duplicated and removed. We could add it to the `remove_duplicate_postmeta` command before the delete, or we could create a separate command for detection only and call it here first. What do you think?

Reply:
Hi @Inwerpsel , As per our discussion on chat I open a [new PR ](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/279) for printing a meta key counts before delete. please have a look.